server/storage/sqlite_test.go

```diff
@@ -17,7 +17,7 @@ func sqlite3Setup(t *testing.T) (*SQLStorage, func()) {
 	tempBaseDir, err := ioutil.TempDir("", "notary-test-")
 	require.NoError(t, err)
 
-	dbStore := SetupSQLDB(t, "sqlite3", tempBaseDir+"test_db")
+	dbStore := SetupSQLDB(t, "sqlite3", tempBaseDir+"/test_db")
```

Comment:
can you use filepath.Join here rather than explicit "/".

Reply:
make sense, Done